jena-base/src/main/java/org/apache/jena/atlas/lib/CacheFactory.java

```diff
@@ -33,7 +33,7 @@
 
     /** Create a null cache */
     public static <Key, Value> Cache<Key, Value> createNullCache() {
-        return new Cache0<>() ;
+        return new CacheGuava<>(0) ;
     }
```

Comment:
You mean see to it that `Cache0` is not used at all, then instead of reimpling it with Guave, just get rid of it entirely?